django_mailbox/models.py

```diff
@@ -642,8 +642,12 @@ def get_body(self):
         if they are encoded as such.
 
         """
+        settings = utils.get_settings()
+
         if self.encoded:
-            return base64.b64decode(self.body.encode('ascii'))
+            return base64.b64decode(
+                self.body.encode(settings['default_charset'], 'replace')
```

Comment:
This change definitely isn't necessary now that I'm re-reviewing my own changes here -- a base64-encoded value will never have non-ascii characters.  I'll remove it  this before the final version.